mlrun/runtimes/nuclio/application/application.py

```diff
@@ -122,6 +122,11 @@ def __init__(
             state_thresholds=state_thresholds,
             disable_default_http_trigger=disable_default_http_trigger,
         )
+
+        # Override default min/max replicas (don't assume application is stateless)
+        self.min_replicas = min_replicas or 1
+        self.max_replicas = max_replicas or 1
```

Comment:
you can do it in the call to `__init__` (lines 91-92)

Reply:
I think its more explicit to have it here. If you do it in the super call it can be easy to miss